### ISSUE TITLE
Training with no half_life data

### ISSUE THREAD

**Issue by AndreiBlindu** [vagarwal87/Xpresso#6]
Hi, I want to retrain your model on custom data where I have paired DNA sequences and gene expression, but no mRNA half-life features.
So, to evaluate the loss in performance due to the lack of half life data, first I wanted to retrain the model on your original data but setting the half-life features to zero, as follows

`
X_trainhalflife = np.zeros(X_trainhalflife.shape, dtype='float32')
`
`
X_validhalflife = np.zeros(X_validhalflife.shape, dtype='float32')
`
`
X_testhalflife = np.zeros(X_testhalflife.shape, dtype='float32')
`

Then, I runned the code without modifying the model's architecture. 
The problem is that the validation loss gets stuck at 1.01833 without further improving after the first epoch.

The same happens if I remove the half-life features input from the model.

I tried using the following configurations:
*on cluster*
- python 3.7.12
- TF version 1.15.0
- Keras version 2.2.4 
- run on CPU due to incompatibility with my cuda driver (I cannot change without sudo permissions)

*on Google Colab notebook*
- python 3.10.12
- TF version 2.17.1
- Keras version 3.5.0
- run on T4 GPU

*on my local Mac*
- python 3.10.11
- TF version 2.15.0
- Keras version 2.15.0
- run on CPU

Do you have any suggestion for training the model without having mRNA half-life features?

Thanks!
